core/psa_crypto_core.h

```diff
@@ -17,18 +17,6 @@
 #include "mbedtls/threading.h"
 #endif
 
```

Comment:
Please forward-port the changelog entry as well. TF-PSA-Crypto 1.0 is the successor of the crypto part of Mbed TLS 3.6, so it will come with a changelog where the first entry describes changes with respect to Mbed TLS 3.6.0.